packages/web-frontend/src/screens/desktop/RootScreen/index.js

```diff
@@ -27,10 +28,11 @@ import { OverlayContainer } from '../../../utils';
 import { TOP_BAR_HEIGHT } from '../../../styles';
 import './desktop-styles.css';
 
-export const RootScreen = ({ enlargedDialogIsVisible }) => {
+export const RootScreen = ({ enlargedDialogIsVisible, isLoading }) => {
   return (
     <div>
       {/* The order here matters, Map must be added to the DOM body after FlexContainer */}
+      <LoadingScreen isLoading={isLoading} />
```

Comment:
I'd suggest we put it further down, so that it is later in the render stack and would obscure the overlay/modal etc.

Reply:
I can move it if you like but the loading screen style has `zIndex: Number.MAX_SAFE_INTEGER` so it's always rendered on top regardless. I put it at the top to make that layout consistent between `mobile` and `desktop`

![image](https://user-images.githubusercontent.com/69437175/117371658-5e3e0500-af1c-11eb-8c16-14b6182a6b2b.png)

Reply:
Hmmm...I still think it's nice to telegraph the fact it'll be over the top of the modal to other developers